src/main/java/org/inferred/freebuilder/EqualsAndHashCode.java

```diff
@@ -0,0 +1,23 @@
+package org.inferred.freebuilder;
```

Comment:
Hi, @alicederyn. Let me know if this is a proper place for the new annotations. I was thinking any annotation a developer would need to use (as today thats only `@FreeBuilder`) should exist in the top level package as well. As of now I'll update the jar-footprint to match, but if you believe differently I'm happy to change things.

Reply:
I would prefer top-level, readable annotations: `@IgnoredByEquals` and `@NotInToString`. There's no need to mention HashCode in the type name as it's a simple consequence of the contract of hashcode; the JavaDoc is sufficient to call out the implication. Calling the annotation just "Exclude" means you can't statically import it and have it remain readable, so it will always be "@EqualsAndHashCode.Exclude", which is less immediately clear.

Reply:
Sounds good to me.